opensquirrel/exporter/quantify_scheduler_exporter.py

```diff
@@ -29,19 +32,32 @@ def __init__(self, qubit_register_name: str):
         self.schedule = quantify_scheduler.Schedule("Exported OpenSquirrel circuit")
 
     def visit_measure(self, g: Measure) -> None:
-        self.schedule.add(quantify_scheduler_gates.Measure(self._get_qubit_string(g.qubit)))
+        qubit_index = g.qubit.index
+        self.schedule.add(
+            quantify_scheduler_gates.Measure(
+                self._get_qubit_string(g.qubit),
+                acq_channel=qubit_index,
+                acq_index=qubit_index,
+                acq_protocol="ThresholdedAcquisition",
+            )
+        )
+        return
 
     def visit_bloch_sphere_rotation(self, g: BlochSphereRotation) -> None:
+        # Note that when adding a rotation gate to the quantify-scheduler Schedule, that there exists an ambiguity with
+        # how quantify-scheduler will store an angle of 180 degrees. Depending on the system the angle may be stored as
+        # either 180 or -180 degrees. This ambiguity has no physical consequences, but may cause the test of the
+        # quantify-scheduler exporter to fail.
```

Comment:
```suggestion
        # Note that when adding a rotation gate to the Quantify-scheduler Schedule,
        # there exists an ambiguity with how Quantify-scheduler will store an angle of 180 degrees.
        # Depending on the system the angle may be stored as either 180 or -180 degrees.
        # This ambiguity has no physical consequences, but may cause the exporter test fail.
```

I removed a `that`, because you already start with a `Notice that`.